aexpect/client.py

```diff
@@ -1093,7 +1093,7 @@ def cmd_output_safe(self, cmd, timeout=60):
                 o += self.read_up_to_prompt(0.5)
                 success = True
                 break
-            except ExpectError, e:
+            except ExpectError as e:
```

Comment:
That's a good point as I don't remember a discussion about aexpect minimum Python version requirements.  At least in Travis, it's configured to test under Python 2.6 and Python 2.7.  @lmr any suggestions here?

Still, how does aexpect influence Avocado-VT (which shares the same Avocado Python version requirements) Python requirements?  I mean, Avocado-VT and Avocado require Python 2.6, so as I see it, there's no link between aexpect + Python 2.4 and Avocado-VT.